components/add_new_cv_form.tsx

```diff
@@ -92,8 +97,40 @@ export default function AddNewCvForm({ id }: Props) {
   }
 
   async function upsertProjects(values: any, cvData: any) {
-    values.cv_id = cvData[0].id;
-    return supabase.from("projects").upsert(values);
+    const cv_id = cvData[0].id;
+
+    const projectValues = {
+      id: values.projects.id,
+      name: values.projects.name,
+      description: values.projects.description,
+      field: values.projects.field,
+      team_size: values.projects.team_size,
+    };
+
+    const { data: projects, error: projectError } = await supabase
+      .from("projects")
+      .upsert(projectValues);
+
+    setServerErrorMessage(projectError ? projectError.message : "");
```

Comment:
How will we know what error occurred here if this server error message is overwritten with the next server error message?